web/src/layouts/ProgramManageLayout.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+  <v-skeleton-loader
+    v-if="isLoading || isNil(program)"
+    type="table"
+  />
+  <TabCard
+    v-else
+    :tabs="tabs"
+  >
+    <router-view />
+  </TabCard>
```

Comment:
I suggest moving `<router-view />` outside of the tab as the tab cards default slot is inside a `v-tabs-window` and this might have a weird effect when the user switches tabs.